src/App.js

```diff
@@ -1,12 +1,108 @@
+            >+</button></h4>
+
+            <h4> 🥝 Kiwi's      <button id="fruit"
+                type="button" onClick={() => setKiwiCounter(kiwiCounter -1)}
+            >-</button>  {kiwiCounter} <button id="fruit"
+                type="button" onClick={() => setKiwiCounter(kiwiCounter +1)}
+
+            >+</button></h4>
+
+            <h3>fruitbasket <button id="fruitbasket" type="reset" name="reset fruit">RESET</button></h3>
+            </div>
+            <h1>Fruit Bestelformulier</h1>
+            <form onSubmit={handleSubmit(onSubmit)}>
+                <br/>voornaam<br/>
+                <input id="lab" ref={register({required: true})} type="text" name="fullName" placeholder="voornaam" />
+                {errors.fullName && <p>❌naam is verplicht</p>}
+                <br/>achternaam<br/>
+                <input id="lab" ref={register({required: true})} type="text" name="lastName" placeholder="achternaam"/>
+                {errors.lastName && <p>❌achternaam is verplicht</p>}
+                <br/>leeftijd<br/>
+                <input id="lab" ref={register({required:true, min: 18})} type="text" name="age" placeholder="leeftijd"/>
+                {errors.age && <p>❌leeftijd van 18 jaar is verplicht</p>}
+                <br/>postcode<br/>
+                <input id="labe" ref={register({required: true, validate:/(^[0-9]{4})([A-Za-z0-9]{2}$)?/})} type="text" name="city" placeholder="postcode"/>
+                {errors.city && <p>❌postcode is verplicht: 0000ab</p>}
+                <br/>huisnummer<br/>
+                <input id="labe" ref={register({required: true, validate: /^\d+[a-zA-Z]*$/})} type="text" name="housenumber" placeholder="huisnummer"/>
+                {errors.housenumber && <p>❌huisnummer is verplicht incl. toevoeging i.v.t.</p>}
+                <br/>
+
+                <h6>Bezorgfrequentie</h6>
+                <label htmlFor="field-frequency">
+                    <input ref={register} type="radio" name="frequency" id="field-weekly" value="weekly"/>
+                    elke week
+                </label><br/>
+                <label htmlFor="field-frequency">
+                    <input ref={register} type="radio" name="frequency" id="field-anotherweek" value="anotherweek"/>
+                    om de week
+                </label><br/>
+                <label htmlFor="field-frequency">
+                    <input ref={register} type="radio" name="frequency" id="field-everymonth" value="everymonth"/>
+                    elke maand
+                </label><br/>
+                <label htmlFor="field-frequentie">
+                    <input ref={register} type="radio" name="frequency" id="field-somethingother" value="somethingother"/>
+                    anders
+                </label>
+                <label>
+                    {watchAnders && <input type="text" name="remark bezorgfrequentie" placeholder="bezorgfrequentie" ref={register({})} />}
```

Comment:
Het input veld staat er nu altijd, ongeacht welke radiobutton is aangetikt. Je zou hier `watchAnders === "somethingother"` moeten hebben staan.

Reply:
Bedankt hier ben ik naar op zoek geweest hoe ik dit kon toepassen.